src/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  person: Person,
+};
+
+export const PersonInfo: React.FC<Props> = ({
+  person,
+
+}) => {
+  return (
+    <div className="dropdown-item">
```

Comment:
if you dont have any logic in your component and just have the jsx, then you can return it without word `return`

```
}) => (
  <div className="dropdown-item">
```

check all your components